frontend/src/components/layouts/AnalyticsHeader.tsx

```diff
@@ -66,26 +66,13 @@ const AnalyticsHeader: React.FC<IHeaderProps> = (props: IHeaderProps) => {
     i18n.changeLanguage(lng);
   };
 
-  const getSelectLabel = (
-    label: string,
-    version: string,
-    reportingEnabled: boolean
-  ) => {
-    if (version === '' || version.startsWith('v1.0')) {
+  const getSelectLabel = (label: string, analysisStudioEnabled: boolean) => {
+    if (!analysisStudioEnabled) {
       return `${label} (${t('analytics:labels.pipelineVersionNotSupport')})`;
```

Comment:
Might we update the label `pipelineVersionNotSupport` to `The Analytics Studio is not available for this application because of an incompatible pipeline version or a disabled reporting feature.`?